### ISSUE TITLE
Actualizacion periodica

### ISSUE THREAD

**Issue by tryagain1111** [PucelaBits/websegura#228]
En la pagina dice "(Las notas se actualizan cada 3 días)" pero algunas de los Score de las webs en Observatory Frifefox no coincide con lo que aparece en Pucelabits.
¿Saben realmente cada cuanto se refresca y si es automatico o bajo demanda?

**Comment by adrm** [comment]
El mensaje no es suficientemente completo. Los datos se actualizan como máximo cada tres días. Cada tres días las webs entran en la cola de webs pendientes de actualizar y se actualizan si hay hueco en la cola, así que durante algunos días puede que haya discrepancia con Observatory. Debería actualizarse en unos pocos días.

**Comment by tryagain1111** [comment]
Gracias adrm,
¿Puede ser que entonces duren mas de 2 semanas en refrescarse ese Score y que se vea tanto ObservatoryMozilla como en la web PucelaBits el mismo?
Por otro lado, ¿ante 2 puntuaciones iguales que orden se sigue?
Gracias

**Comment by adrm** [comment]
Las puntuaciones en observatory se refrescan cuando alguien lo solicita allí, o cuando nosotros lo solicitamos por la api.

Puede que, depende cómo se ordene la cola, haya páginas pendientes de actualización que se queden atrás y no se vayan actualizando. Si es así, aceptamos PR mejorando la gestión de la cola.

Es posible que se solicite un análisis desde Observatory y que pasen unos días hasta que nuestra web coja ese resultado y lo mostremos actualizado.

Con el orden, ¿te refieres a la hora de ordenar la lista de páginas? Si hay dos con score igual creo recordar que se ordenan por orden alfabético.

**Comment by tryagain1111** [comment]
Sí, las puntuaciones a nivel de Observatory las veo que cambian al vuelo, pero el refresco de Pucelabits con esa gestion de la cola (que no acabo de ver) es muy lento, es mas de 10 días lo que lleva a publicar e informar de una información erronea.
